pennylane/measurements/classical_shadow.py

```diff
@@ -343,20 +357,25 @@ class ShadowExpvalMP(MeasurementTransform):
     Please refer to :func:`shadow_expval` for detailed documentation.
 
     Args:
-        args (tuple[Any]): Positional arguments passed to :class:`~.pennylane.measurements.MeasurementProcess`
-        H (:class:`~.pennylane.Hamiltonian` or :class:`~.pennylane.operation.Tensor`): Observable
-            to compute the expectation value over.
+        H (Operator, Sequence[Operator]): Operator or list of Operators to compute the expectation value over.
```

Comment:
You don't accept Hamiltonian object anymore?

Reply:
`Hamiltonian` and `Tensor` are both `Operator`'s.  But it also accepts other operators types as well.